src/listener/can_open_listener.py

```diff
@@ -86,10 +86,13 @@ async def async_network_loop(self):
         ''' Function to loop the through all nodes asynchronously.
             Loops forever.
         '''
-        while True:
-            self.listen_to_network()
-            # Sleeps 0.1 seconds
-            await asyncio.sleep(0.1)
+        try:
+            while True:
```

Comment:
Moet deze try/except ook niet IN de while loop?

Reply:
Over het algemeen maakt het niet veel uit. https://www.quora.com/Should-exception-handling-go-inside-or-outside-a-loop 
Maar bij onze situatie wel, stel je hebt de exception. dan stopt het met de while loop in de huidige situatie.
Als try/except binnen de loop zit dan gaat het toch door itereren en dat is niet gewenst.